### ISSUE TITLE
Request: Convert to tile

### ISSUE THREAD

**Issue by Terrox** [mcglincy/dungeondraw-foundry-vtt#121]
I'd often want to draw up some rooms on-the-fly but my dungeon maps are gigantic, so I can't really see a way to get my new room into place without it being a tile.

**Comment by mcglincy** [comment]
hi, sorry for the delay responding. Can you help me understand what exactly you're trying to do? It sounds like you have an existing map image for the scene background, but you want to use Dungeon Draw to just draw a room or two to show with the existing non-Dungeon Draw map?

**Comment by Terrox** [comment]
yes, using dungeon draw to make smaller segments on a layer above the background so I can reposition, sort of modular.

**Comment by oskarnp314** [comment]
#159 implements the desired behaviour here. 
@Cussa I made that feature for exactly for use with mass edit/token attacher.
